core/src/test/java/org/apache/iceberg/rest/RESTCatalogAdapter.java

```diff
@@ -155,6 +191,14 @@ public static Pair<Route, Map<String, String>> from(HTTPMethod method, String pa
 
       return null;
     }
+
+    public Class<? extends RESTRequest> getRequestClass() {
```

Comment:
Minor: we don't use `get` in methods names since it isn't adding anything. These should be `requestClass` and `responseClass`